src/emulator/auth/operations.ts

```diff
@@ -1011,6 +1041,31 @@ export function setAccountInfoImpl(
   });
 }
 
+function sendOobForEmailReset(state: ProjectState, oldEmail: string, newEmail: string, url: URL) {
+  const mode: string = "recoverEmail";
+
+  // Encode the new email with the OOB code. The old email is received as a URL parameter.
+  const { oobCode, oobLink } = state.createOob(newEmail, "RECOVER_EMAIL", (oobCode) => {
```

Comment:
These seem copy pasted from `sendOobCode`. Can we extract a reusable function?

Reply:
Done.